src/device.rs

```diff
@@ -135,6 +143,21 @@ impl<SPI, NCS, E, EO> Device for SpiDevice<SPI, NCS>
         self.ncs.set_high().map_err(|a| SpiError::NCSError(a))?;
         Ok(())
     }
+
+    fn write_many(&mut self,
+                  reg: Register,
+                  buffer: &[u8])
+                  -> Result<(), Self::Error> {
+        if buffer.len() > 16 {
```

Comment:
Would be nice to handle this in compile-time, but I guess we have to wait for const generics or re-introduce genericarray crate to do that :-(

Reply:
So, are you ok to let this as is?

Reply:
Yes!